src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -8615,9 +8459,10 @@ public void BreakingChange_ReturnOutByRef()
         }
 
         // Breaking change in C#11: Instance method on ref struct instance may capture unscoped ref or in arguments.
+        // No longer a breaking change
```

Comment:
Consider deleting comments, to avoid confusion.

Reply:
```
// increment this counter when editing the comment: 3
```